rogue/src/terrain_spawners/water.rs

```diff
@@ -105,7 +105,7 @@ pub fn shallow_water(ecs: &mut World, x: i32, y: i32, fgcolor: RGB, bgcolor: RGB
             glyph: rltk::to_cp437('~'),
             fg: fgcolor,
             bg: bgcolor,
-            order: 3,
```

Comment:
Just a small oppurtunistic bugfix.

Grass and water was set to the same render order, and this seemed to lead to non-deterministic ordering, sometimes the grass would render, sometimes it would be masked by the water. We want to see the sweet, sweet grass.